src/api/utils/InterceptorManager.js

```diff
@@ -0,0 +1,38 @@
+class InterceptorManager {
+  handlers = [];
+
+  use(fullfill, failure) {
+    this.handlers.push({
+      fullfill,
+      failure,
+    });
+  }
+
+  remove(index) {
```

Comment:
Not sure if we need this 🤔 , in which scenario do you imagine using it?

Reply:
Right now, on any, but it seems necessary for api completeness, otherwise you would have `.use` to add middlewares, but no way to remove them. Seemed kind of weird